docs/section9/Rule6-mno.md

```diff
@@ -0,0 +1,58 @@
+
+# Lighting - Rule 6-mno
+
+**Rule ID:** 6-mno  
+**Rule Description:** ILP for the proposed building is determined as designed/as exisiting or determined using BAM for lighting system not designed and not exisiting. And the ILP for the baseline building is determined using values in Table G3.7 (Note XC, is this only applied if the building goes for space-by-space method?)  
```

Comment:
@xingcx I agree with Jason's comment. This might help you think through the required RDS 

1. 6mno- Proposed - as designed/ as existing/not designed.  BEM tools usually dont have a way to identify lighting as designed, as existing or not designed. So I think we need to check if the 'lighting_building_area_type' object at the building segment level is used. If yes, we can verify the LPDs against the prescribed values in Table 9.5.1. If not, we assume lighting is as-designed, or as-existing and the RCT outcome is of 'Caution- Manual review required'.
2. 6-mno- Baseline. Here the check is only to see if the proposed is using the building area method, in which case the baseline should also be using G3.8, else it should be using G3.7

Reply:
@JasonGlazer - just to clarify, you've mentioned 'Table G3.7 is used for the baseline no mater what the design used.'. However, the baseline should use Table G3.8 if the proposed uses building area method, correct?

Reply:
@supriyagoel in Table G3.1 part 6, baseline says: "Interior lighting power in the baseline building design shall be determined using the values in Table G3.7." It does not mention Table G3.8 which is only referenced in G1.2.1(b)(2).

Reply:
@supriyagoel I think if we want to differentiate between as-designed/as existing/not designed we should propose to the Schema WG to add a explicit enumeration for that. Trying to determine it from a "null" value seems to complicated.

Reply:
@JasonGlazer sure, having a data element would be helpful. Lets see what the schema WG says.
Regarding Table G3.8 - I had discussed that with you and Mike T when we discussed the lighting TCDs and I agree with you - App G doesnt not say that Table G3.8 should be used. Then how would one determine the space types for the baseline if the spaces in the proposed are not defined? 
This seems to be an issue in App G.

Reply:
@supriyagoel I think we should use the building area type for the spaces when they are not defined.

Reply:
@JasonGlazer Ok, so i I understand you correctly, we would look at Table G3.7 and find the corresponding space type for the building area type? If yes, then G3.7 doesnt have all a space type called 'Hospita'l; or 'Hotel/Motel' (i do see a courthouse')
adding @mkarpman so that she can weigh in.

Reply:
@JasonGlazer how would be know if lighting in spaces is not defined? Irrespective of whether lighting is specified or not, it should be modeled for both baseline and proposed design. Baseline must always be based on G3.7. For the proposed, unless there is an RMR data element saying that lighting is not actually specified (which is not something that is typically captured in BEM tool in my experience), we probably can't do any tests other than that the modeled LPD is not 0 (which would be a mistake). So perhaps RCT should always produce a warning message to reviewer that if project includes any spaces where lighting is not specified, the LPD in the proposed model must be manually verified to be based on Table 9.5.1. 
@supriyagoel, I would consider incorporating building area types included in table G3.7 within enumeration - e.g., Assisted Living Facility - Chapel

Reply:
@mkarpman we have gone back and forth on the value of adding a data element for is_lighting_designed. Like you said, this is not something that tools capture so there is no value of an element like that. In all cases, the RCT is writing out a caution message for the proposed building (3 cases- lighting as-designed, lighting-as existing, lighting-not designed) because we cannot verify any of these 3 cases. However, we can at least check against G3.7 for lighting in proposed and also say that proposed LPD is equal to/or not equal to G3.7.
We can do what you've recommended for the table.

Reply:
Great minds think alike :-). To confirm, if lighting is not specified, the proposed design should be based on 9.5.1 (BAM in section 9), not G3.7.

Reply:
> Please do not use abbreviations for ILP or BAM
> 
> Table G3.7 is used for the baseline no mater what the design used.
> 
> I think this should be split into two RDS files. One for the proposed portion and the other for the baseline.

@supriyagoel Please confirm if ILP is Interior Lighting Power, and BAM is Building Area Method.

Reply:
I have split this rule into two, and if we reference Addendum af:
https://www.ashrae.org/file%20library/technical%20resources/standards%20and%20guidelines/standards%20addenda/90_1_2019_af_bc_cd_db_20201116.pdf

For Proposed Building

> c. Where lighting neither exists nor is submitted with design documents, lighting shall comply with but not exceed the requirements of Section 9. Where space types are known, lighting power shall be determined in accordance with the Space-by-Space Method. Where space types are not known, lighting Lighting power shall be determined in accordance with the Building Area Method.

For Baseline

> Interior lighting power in the baseline building design shall be determined using the values in Table G3.7. However, where lighting neither exists nor is submitted with design documents, and the proposed design lighting power is determined in accordance with the Building Area Method, the baseline building design lighting power shall be determined in accordance with Table G3.8. Where retail display lighting is included in the proposed building design in accordance with Section 9.6.2(b), the baseline building design retail display lighting additional power shall be equal to the limits established by Section 9.6.2(b) or same as proposed, which ever less.

Let me know if the following flow works:
Rule 6-mno-b, for baseline
Check if the proposed model uses BAM
-> yes, use Table G3.8 (BAM)
-> no, use Table G3.7 (Space-by-Space)

Rule 6-mno-p, for proposed
Check if the proposed model uses BAM
-> yes, compare total modeled wattage to Table G3.8 (BAM)
-> no, the RCT outcome is of 'Caution- Manual review required'.

Also I feel that in the context of Appendix G, Building Area Method references Table G3.8 (Performance Rating Method...Using the Building Area Method) not Table 9.5.1, and Space-by-Space Method references Table G3.7 (Performance Rating Method...Using the Space-by-Space Method) not Table 9.6.1, as the Mandatory Provision G1.2.1(b) references Table G3.7 and only the methodology described in Section 9.6.1, etc. Also Table G3.1-6(l) reference controls credits in Table G3.7. Let me know if this is the right understanding.

Reply:
@xingcx I'm not sure that Addendum AF is relevant since it is changing the requirements of 90.1-2019.

Reply:
@xingcx please make a new pull request for rule 6mno and add link to this discussion.